docs/sources/alerting/unified-alerting/grafana-managed-number-alerts.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "Numeric data Grafana manged alert rules"
-description = "Number data based Grafana manged alert rules"
+title = "Numeric data Grafana managed alert rules"
```

Comment:
How about "Alert rules for numeric data"? The title as-is is a little confusing. If we want to stress that is a Grafana alert, then: "Grafana managed alert rules for numeric data".